### PR TITLE
nix: new package starting at 2.0.4

### PR DESCRIPTION
**Pull request by tristan0x**

### CODE REVIEW

File: var/spack/repos/builtin/packages/nix/package.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
+##############################################################################
```
Comment (healther) [inline var/spack/repos/builtin/packages/nix/package.py]: please update to the new license

Comment (alalazo) [reply var/spack/repos/builtin/packages/nix/package.py]: License has been updated.

**Comment by tristan0x** [comment]
This PR was not supposed to be merged as is. it is still WIP

**Comment by alalazo** [comment]
Ouch, where did it say that? Being only a package I just checked that the recipe were written without clear issues. Can you work on a following PR to complete what is needed?

**Comment by tristan0x** [comment]
yes I will do that.
Tristan Carel

**Pull request merged**